templates/tutorialv2/view/content.html

```diff
@@ -122,7 +122,7 @@ <h2 class="subtitle">
         {% if content_helps %}
             <div class="content-wrapper">
                 <div class="alert-box info">
-                    {% if content.authors.count > 1 %}
+                    {% if content.redacting_authors.count > 1 %}
```

Comment:
Étant donné que `redacting_authors` est une liste, je ne crois pas que `.count` fonctionne. Si ça ne fonctionne pas, alors ceci devrait fonctionner : 
```suggestion
                    {% if content.redacting_authors|length > 1 %}
```